src/types/grid.ts

```diff
@@ -56,6 +56,10 @@ export type SupabaseGridProps = {
    * show add row button if available
    */
   onAddRow?: () => void;
+  /**
+   * show delete row button if available
+   */
+  onDeleteRow?: () => void;
```

Comment:
Delete row should be handled by the grid. You can take a look [here](https://github.com/supabase/grid/blob/9d6d64b4466f1ed54a4ae61f98801ca5fb1ca7ec/src/components/menu/RowMenu.tsx#L21-L34)

Reply:
@phamhieu is this what you suggested?